disnake/http.py

```diff
@@ -2002,6 +2003,37 @@ def get_guild_scheduled_event_users(
         )
         return self.request(route, params=params)
 
+    # Welcome screens
+
+    def get_guild_welcome_screen(
+        self, guild_id: Snowflake
+    ) -> Response[welcome_screen.WelcomeScreen]:
+        r = Route("GET", "/guilds/{guild_id}/welcome-screen", guild_id=guild_id)
+        return self.request(r)
+
+    def edit_guild_welcome_screen(
+        self,
+        guild_id: Snowflake,
+        enabled: bool = None,
+        welcome_channels: List[welcome_screen.WelcomeScreenChannel] = None,
+        description: str = None,
+        reason: str = None,
+    ) -> Response[welcome_screen.WelcomeScreen]:
+
+        payload: Dict[str, Any] = {}
+
+        if description is not None:
+            payload["description"] = description
```

Comment:
This way it's not possible to remove the description by passing `None`. According to docs, all parameters here are nullable: https://discord.com/developers/docs/resources/guild#modify-guild-welcome-screen

Reply:
I'm concerned about using MISSING in http.py, as I haven't seen it been used anywhere else, so I'll have to figure out a different way to write this method's definition.

Reply:
https://github.com/DisnakeDev/disnake/pull/339/commits/cf6aded359eaa078cfdcaa5997022dcd2fa8287b